tests/failpoints/cases/test_replica_read.rs

```diff
@@ -292,19 +292,19 @@ fn test_read_after_cleanup_range_for_snap() {
     assert_eq!(cluster.leader_of_region(region.get_id()).unwrap(), p1);
     cluster.stop_node(3);
     (0..10).for_each(|_| cluster.must_put(b"k1", b"v1"));
-    // Ensure logs are compacted.
-    must_truncated_to(cluster.get_engine(3), r1, 8);
+    // Ensure logs are compacted, then node 1 will send a snapshot to node 3 later
+    must_truncated_to(cluster.get_engine(1), r1, 8);
```

Comment:
How can it pass before?

Reply:
I guess it is because the logs are compacted quickly.